documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -3,16 +3,26 @@
 // assembly-kafka-connect.adoc
 
 [id='proc-deploying-kafkaconnector-{context}']
-= Deploying a `KafkaConnector` resource to Kafka Connect
+= Deploying the example `KafkaConnector` resource
 
-This procedure describes how to deploy the example `KafkaConnector` to a Kafka Connect cluster.
+Strimzi includes an example `KafkaConnector` in `examples/connect/source-connector.yaml`. 
+This creates a basic `FileStreamSourceConnector` instance that sends each line of the Kafka license file (an example file source) to a single Kafka topic. 
 
-The example YAML will create a `FileStreamSourceConnector` to send each line of the license file to Kafka as a message in a topic named `my-topic`.
+You can create a `FileStreamSinkConnector` by changing the configuration of the example `KafkaConnector`.
+
+The following procedure illustrates how to configure Kafka Connect connectors by using the `source-connector.yaml` file as an example. 
+
+[NOTE]
+====
+The `FileStreamSourceConnector` and `FileStreamSinkConnector` are not intended to be run in containers. 
+In a production environment, you should prepare container images containing the Kafka Connect connectors that you want to use.
```

Comment:
Please can we link to the procedure for doing this?

Reply:
We have the automatic option now. https://strimzi.io/docs/operators/master/deploying.html#using-kafka-connect-with-plug-ins-str